tools/purge_ems.rb

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env ruby
+enviroment = File.expand_path('../config/environment', __dir__) + ".rb"
+require enviroment if File.exist?(enviroment)
+
+if !defined?(Vmdb::Application)
+  warn "This script must exist in the tools directory or can live elsewhere but needs to be run via rails runner path/to/purge_ems.rb --id 1234 --follow true"
+  exit 1
+end
```

Comment:
This is what all (many?) of the other tools scripts do instead.

```suggestion
require File.expand_path("../config/environment", __dir__)
```

Reply:
read above

Reply:
Yeah I don't think either of those are actually a thing.  The file can be copied into the tools dir and you don't need to run with a literal rails runner if you require the environment file.

Reply:
I love problems that can be completely removed 😄